src/styles/stars.css

```diff
@@ -0,0 +1,41 @@
+.card__block-stars {
```

Comment:
You have a file called like starts.css, and you have a lot of selectors(.card__block-stars, .card__block-price, .card__text) here that do not belong to the stars, but are part of the card. Move it to card.css file